esp/esp/users/models/__init__.py

```diff
@@ -1354,6 +1362,7 @@ class TeacherInfo(models.Model, CustomFormsLinkModel):
 
     user = AjaxForeignKey(ESPUser, blank=True, null=True)
     graduation_year = models.CharField(max_length=4, blank=True, null=True)
+    affiliation = models.CharField(max_length=100, blank=True, null=True)
```

Comment:
Ah, got it.

Also, contrary to what the other things in this file do, the correct thing is actually `blank=True` but not `null=True`.  This will also require updating the migration.